safe/impact_function/create_extra_layers.py

```diff
@@ -161,3 +162,31 @@ def create_profile_layer(profiling):
 
     tabular.commitChanges()
     return tabular
+
+
+def create_valid_aggregation(layer):
+    """Create a local copy of the aggregation layer and try to make it valid.
```

Comment:
Maybe good to add in the docstring why we need this (i.e. that geometries may need to be cleaned)